app/views/provider/admin/account/payment_gateways/braintree_blue/edit.html.slim

```diff
@@ -1,3 +1,40 @@
+css:
```

Comment:
@thomasmaas I did that inline because I needed some styling for the divs too look like an input field, could you please help on this PR :)